.github/workflows/container-image.yml

```diff
@@ -38,7 +38,15 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: Build the server image
-      run: make build-server
+      run: |
+        make build-server
```

Comment:
Is there are reason this isn't two run steps? LIke:
```
- name: Build the server image
  make build-server
- name: Export the server image
  podman save samba-container:latest > /tmp/server-image.tar
```

Reply:
Make sense. Updated in every other occurrence.